test/test_runner.c

```diff
@@ -187,15 +187,22 @@ void CB2_TestRunner(void)
                 }
                 break;
             case TEST_RESULT_PASS: result = "PASS"; break;
-            case TEST_RESULT_SKIP: result = "SKIP"; break;
+            case TEST_RESULT_SKIP:
+                result = "ASSUMPTION_FAIL";
+                color = "\e[33m";
+                break;
             case TEST_RESULT_INVALID: result = "INVALID"; break;
             case TEST_RESULT_ERROR: result = "ERROR"; break;
             case TEST_RESULT_TIMEOUT: result = "TIMEOUT"; break;
             default: result = "UNKNOWN"; break;
```

Comment:
At this point, I'd probably just format all the cases on multiple lines :)

Reply:
are there other reasons tests might be skipped? For readability I would probably rephrase it to `TEST_RESULT_ASSUMPTION_FAIL` or something along those lines.

Reply:
Yeah, I didn't find other causes for skipping, so it would make sense.